app/src/main/java/org/wikipedia/page/PageFragment.java

```diff
@@ -965,25 +963,6 @@ private int getForegroundTabPosition() {
     private void setupMessageHandlers() {
         linkHandler = new LinkHandler(requireActivity()) {
             @Override public void onPageLinkClicked(@NonNull String anchor, @NonNull String linkText) {
-                dismissBottomSheet();
-                if (anchor.startsWith("cite_note")) {
-                    // It's a link to another reference within the page.
```

Comment:
The purpose of this was to handle clicking on references *within* the text of a reference. How will this be handled now?  For an example, go to the [[Earth]] article, scroll to the "Name and etymology" section, tap the [n 7] reference in the first sentence, and tap the [23] reference within the reference.

Reply:
This is done @dbrant . Since references endpoint will be deprecated, for references withing reference content, it only scrolls to the place where the reference is.